library/core/src/num/nonzero.rs

```diff
@@ -1543,33 +1543,33 @@ macro_rules! nonzero_integer_signedness_dependent_methods {
         #[rustc_const_unstable(feature = "isqrt", issue = "116226")]
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
-        #[inline]
```

Comment:
I think I made an error there. I've restored it to `#[inline(always)]` because with your request for it to call the `uint_macros` `fn isqrt`, [it inlines very well](https://github.com/rust-lang/rust/pull/128166#discussion_r1722390704).

Reply:
Awesome great to eliminate that call!